packages/fxa-settings/src/components/Settings/PageRecoveryPhoneRemove/en.ftl

```diff
@@ -0,0 +1,11 @@
+## PageRecoveryPhoneRemove
+## Users reach this page from account settings when they want to remove a backup phone number.
+
+recovery-phone-remove-header = Remove recovery phone number
+# Variables:
+#   $formattedFullPhoneNumber (String) - the user's full phone number
+settings-recovery-phone-remove-info = This will remove <strong>{ $formattedFullPhoneNumber }</strong> as your backup recovery phone.
+settings-recovery-phone-remove-recommend = We recommend you keep this method because it’s easier than saving backup authentication codes.
+settings-recovery-phone-remove-recovery-methods = If you delete it, make sure you still have your saved authentication codes. <linkExternal>Compare recovery methods</linkExternal>
```

Comment:
I could have localized "Compare recovery codes" separately, but I thought it was nice also to have the context of the sentence prior.

Reply:
I agree, its nice to have the context. 
(Out of sheer paranoia - did you mean `Compare recovery methods` like in the string?)

Reply:
👍
(Yes sorry, that's definitely what I meant!)

Reply:
Reacting to @vpomerleau 's comment below - if we want to leave the string as-is, we should at least have a comment indicating that "saved authentication codes" are referring to previously saved backup authentication codes.